server/importer.js

```diff
@@ -323,7 +323,7 @@ var async = require('async'),
 			alreadyImportedAllPosts = false;
 			alreadyImportedAllVotes = false;
 			alreadyImportedAllBookmarks = false;
-		} else if (alreadyImportedAllVotes) {
+		} else if (areVotesDirty) {
```

Comment:
this whole check function is growing out of hand :P - i have something else in mind - hopefully i will get around to do it after nodebb 1.0 is out